src/index.html

```diff
@@ -6,7 +6,440 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header">
+      <a href="#home" class="page__headerLogo">
+        <img
+          src="images/logo.png"
+          alt="Logo imagine"
+          class="img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="Apple" class="nav_links is-active">Apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="Samsung" class="nav_links">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="Smartphones" class="nav_links">Smartphones</a>
+          </li>
+
+          <li class="nav_item">
+            <a 
+              href="Laptops&Computers"
+              class="nav_links"
+              data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a href="Gadgets" class="nav_links">Gadgets</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="Tablets" class="nav_links">Tablets</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="Photo" class="nav_links">Photo</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="Video" class="nav_links">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div 
+        class="container"
+        data-qa="card"
+      >
+        <img 
+          src="images/imac.jpeg"
+          alt="Apple A1419 iMac 27 Retina 5K Monoblock "
+          class="container__img"
+        >
+        <p class="container__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="container__productNumber">
+          Product code: 195434
+        </p>
+
+        <div class="container__stars">
+
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
```

Comment:
no spaces between non-siblings
```suggestion
        <div class="container__stars">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```